test/integration/smoke/test_kubernetes_clusters.py

```diff
@@ -81,6 +82,20 @@ def setUpClass(cls):
                                       "cloud.kubernetes.service.enabled",
                                       "true")
                 cls.restartServer()
+            if cls.hypervisor.lower() == 'vmware':
+                cls.create_full_clone = Configurations.list(cls.apiclient, name="vmware.create.full.clone")[0].value
+                if cls.create_full_clone not in ["true", True]:
+                    Configurations.update(cls.apiclient,
+                                          "vmware.create.full.clone",
+                                          "true")
+                allStoragePools = StoragePool.list(
+                    cls.apiclient
+                )
+                for pool in allStoragePools:
+                    Configurations.update(cls.apiclient,
+                                          storageid=pool.id,
+                                          name="vmware.create.full.clone",
+                                          value="true")
```

Comment:
I see the same code below, can you externalise this in a separate method?

Reply:
Will do @DaanHoogland